packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -148,13 +148,11 @@ export class VerticalStackedBarChartBase extends React.Component<
           id={this._calloutId}
           {...this.props.calloutProps}
         >
-          {
-            this.props.onRenderCalloutPerStack
-              ? this.props.onRenderCalloutPerStack(this.state.stackCalloutProps)
-              : this.props.onRenderCalloutPerDataPoint
-                ? this.props.onRenderCalloutPerDataPoint(this.state.dataPointCalloutProps, this._renderCallout)
-                : this._renderCallout(this.state.dataPointCalloutProps)
-          }
+          {this.props.onRenderCalloutPerStack
```

Comment:
Let's revert these changes too

Reply:
somehow commit always change formatting this file... I don't know how to fix it.

Reply:
with merge latest master it get fixed